framework/log/SyslogTarget.php

```diff
@@ -40,13 +46,20 @@ class SyslogTarget extends Target
         Logger::LEVEL_ERROR => LOG_ERR,
     ];
 
+    /**
+     * @inheritdoc
+     */
+    public function init()
+    {
+         $this->options = LOG_ODELAY | LOG_PID;
```

Comment:
Wait, this would prevent setting your own options. Should be:

```php
if ($this->options === null) {
    $this->options = LOG_ODELAY | LOG_PID;
}
```